src/docfx/config/ConfigLoader.cs

```diff
@@ -154,8 +157,11 @@ private static (List<Error>, JObject) ExtendConfigs(JObject config, string docse
                 }
             }
 
-            JsonUtility.Merge(result, config);
-            return (errors, result);
+            // to keep the line info of config
```

Comment:
may we extract this as a method?

Reply:
Thought it was not a good idea to put the workaround into lib, but maybe it's ok